From 4cdfef3fedf641a551d97547c9d0d82693ec9bec Mon Sep 17 00:00:00 2001
From: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
Date: Wed, 22 Nov 2017 11:32:31 +0800
Subject: [PATCH] armv7:dts:ls1021a-iot:add sigfox

add sigfox spi device on ls1021a-iot.dts to support sigfox device.

Signed-off-by: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
---
 arch/arm/boot/dts/ls1021a-iot.dts | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/ls1021a-iot.dts b/arch/arm/boot/dts/ls1021a-iot.dts
index 11b18a5..6cf78ac 100644
--- a/arch/arm/boot/dts/ls1021a-iot.dts
+++ b/arch/arm/boot/dts/ls1021a-iot.dts
@@ -158,6 +158,22 @@
 	};
 };
 
+&dspi0 {
+	bus-num = <0>;
+	status = "okay";
+	num-cs = <4>;
+	is-decoded-cs = <0>;
+	spidev@0 {
+		compatible = "rohm,dh2228fv";
+		reg = <0x3>;
+		spi-max-frequency = <4000000>;
+		fsl,spi-cs-sck-delay = <200000>;
+		fsl,spi-sck-cs-delay = <200000>;
+		#address-cells = <1>;
+		#size-cells = <1>;
+	};
+};
+
 &i2c1 {
 	status = "disabled";
 };
-- 
2.14.1

